xarray/core/dataarray.py

```diff
@@ -3358,9 +3358,12 @@ def map_blocks(
         ...     clim = gb.mean(dim="time")
         ...     return gb - clim
         >>> time = xr.cftime_range("1990-01", "1992-01", freq="M")
+        >>> month = xr.DataArray(time.month, coords={"time": time}, dims=["time"])
```

Comment:
Hopefully this is the best way to include this coordinate. Just doing `DataArray(..., dims=["time"], coords={"time": time, "month": time.month})` raised with

```pytb
ValueError: coordinate month has dimensions ('month',), but these are not a subset of the DataArray dimensions ['time']
```